modules/conflict_resolver/php/conflict_resolver.class.inc

```diff
@@ -183,11 +183,14 @@ class Conflict_Resolver extends \NDB_Menu_Filter_Form
      */
     function _setupVariables()
     {
+        // Rest the filter because this is broken with react modules
+        //$this->_resetFilters();
+
         $config =& \NDB_Config::singleton();
 
         // set the class variables
-        $useProjects = $config->getSetting("useProjects");
-        if ($useProjects === "false") {
+        $useProjects = $config->getSetting('useProjects');
+        if ($useProjects === 'false') {
```

Comment:
are we still using tpl_data if there's no smarty template?

Reply:
You're right, I removed the tpl_data with latest commit.